filebeat/processor/add_kubernetes_metadata/matchers.go

```diff
@@ -24,13 +24,16 @@ const LogPathMatcherName = "logs_path"
 
 type LogPathMatcher struct {
 	LogsPath string
+	ResourceType string
 }
 
 func newLogsPathMatcher(cfg common.Config) (add_kubernetes_metadata.Matcher, error) {
 	config := struct {
 		LogsPath string `config:"logs_path"`
+		ResourceType string `config:"resource_type"`
```

Comment:
this config starts to get complicated, I guess we can live with this how it is now, but I will try to give it a thought on how to simplify this mechanism. Not a blocker for this PR :)